spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -877,6 +877,25 @@ def stub_uninteresting_portions_of_install_puppetlabs_dev_repo!
       subject.install_puppet_agent_dev_repo_on( host, opts )
     end
 
+    it 'runs the correct install for osx platforms (newest link format)' do
+      platform = Object.new()
+      allow(platform).to receive(:to_array) { ['osx', '10.9', 'x86_64', 'mavericks']}
+      host = basic_hosts.first
+      host['platform'] = platform
+      opts = { :version => '0.1.0' }
+      allow( subject ).to receive( :options ).and_return( {} )
+      copied_path = "#{win_temp}\\puppet-agent-x64.msi"
```

Comment:
according to my testing, the lines including `allow( subject ).to receive( :options )...` and below are all extraneous copy-pasta that doesn't effect the outcome of the tests.

These can definitely be removed, but at this point, I'd rather let the Jenkins run finish (since they take over an hour), and either come back to fix this in a separate cleanup PR (if run is successful), or put them in the changes to fix the Jenkins failure.  Does that work for you?